kaldb/src/test/java/com/slack/kaldb/logstore/opensearch/OpenSearchAdapterTest.java

```diff
@@ -44,7 +47,9 @@ public String getType() {
         };
 
     openSearchAdapter.buildAggregatorUsingContext(
-        unknownAgg, logStoreAndSearcherRule.logStore.getSearcherManager().acquire());
+        unknownAgg,
+        logStoreAndSearcherRule.logStore.getSearcherManager().acquire(),
+        new MatchNoDocsQuery("testing"));
```

Comment:
You could also pass `null` in these calls as long as the test isn't attempting a min/max agg.